macos/src/main/java/com/github/weisj/darklaf/platform/macos/JNIDecorationsMacOS.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Jannis Weis
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.github.weisj.darklaf.platform.macos;
+
+import com.github.weisj.darklaf.platform.NativeUtil;
+import com.github.weisj.darklaf.util.SystemInfo;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class JNIDecorationsMacOS {
+
+    private static final Logger LOGGER = Logger.getLogger(JNIDecorationsMacOS.class.getName());
+    private static boolean supported;
+    private static boolean loaded;
+
+    public static native void installDecorations(final long hwnd);
+
+    public static native void uninstallDecorations(final long hwnd);
+
+    /**
+     * Load the decorations-library if necessary.
+     *
+     * @return true if successful and library wasn't already loaded.
+     */
+    public static boolean updateLibrary() {
+        boolean oldLoaded = loaded;
+        if (!supported) {
+            supported = loadLibrary();
+        }
+        return supported && !oldLoaded;
+    }
+
+    private static boolean loadLibrary() {
+        if (!SystemInfo.isWindows) {
```

Comment:
Surely you mean `if (!SystemInfo.isMac) {` here :)

Reply:
Note: it is probably worth to avoid loading the library in case Java12+ is used.

Reply:
Yes of course :)

Reply:
> Note: it is probably worth to avoid loading the library in case Java12+ is used.
This depends on whether there is a non jni way of setting the title text color.

Reply:
Oh, `title text color` might be an issue indeed.